load-test/bin/undo-generate-pupil-load-test-data.sql

```diff
@@ -0,0 +1,5 @@
+DELETE FROM mtc.mtc_admin.pupilLogonEvent
+DELETE FROM mtc.mtc_admin.[check] WHERE pupil_id > 2
+DELETE FROM mtc.mtc_admin.answer
+DELETE FROM mtc.mtc_admin.pupil WHERE school_id > 6
+DELETE FROM mtc.mtc_admin.school WHERE urn > 89005
```

Comment:
what is significant about urn `89005`?

Reply:
It's the urn of the last school we generate from the migrations.
I can always swap that for `id > 6`

Reply:
please use Ids, as its inline with the rest of the statements

Reply:
@GuyHarwood updated